ruby/ruby/Ice/Proxy.rb

```diff
@@ -10,21 +10,13 @@ module Ice
     # Provide some common functionality for proxy classes
     module Proxy_mixin
         module ClassMethods
-            def inspect
```

Comment:
Removed because as far as I can tell, it didn't do anything.

Reply:
Doesn't this call ProxyInfo::print?

Reply:
We already implement it on the base ObjectPrx class:
https://github.com/zeroc-ice/ice/blob/b38903c31b79dd75d4bf1bd1e3bcdc2f566a0bef/ruby/src/IceRuby/Proxy.cpp#L1236